net-misc/zerotier/files/zerotier-1.14.0-update-time.patch

```diff
@@ -0,0 +1,33 @@
+https://bugs.gentoo.org/938204
+https://github.com/time-rs/time/issues/681
+https://github.com/time-rs/time/pull/671
```

Comment:
It should really be linking to https://github.com/zerotier/ZeroTierOne/issues/2345 (and if one didn't exist I'd probably suggest opening one).